nautobot/circuits/tests/test_filters.py

```diff
@@ -285,6 +285,27 @@ def setUpTestData(cls):
             Site.objects.create(name="Test Site 2", slug="test-site-2"),
             Site.objects.create(name="Test Site 3", slug="test-site-3"),
         )
+        manufacturer = Manufacturer.objects.create(name="Test Manufacturer 1", slug="test-manufacturer-1")
+        devicetype = DeviceType.objects.create(
+            manufacturer=manufacturer,
+            model="Test Device Type 1",
+            slug="test-device-type-1",
+        )
+        devicerole = DeviceRole.objects.create(name="Test Device Role 1", slug="test-device-role-1", color="ff0000")
+        device1 = Device.objects.create(
+            device_type=devicetype,
+            device_role=devicerole,
+            name="TestDevice1",
+            site=sites[0],
```

Comment:
I'm a little surprised/worried that this works - including `status` should be mandatory for Device creation, I believe.

Reply:
Status isn't mandatory. It can be null. It is more correct to include it however, but not necessary unless it is needed for a test case.

Reply:
Interesting. The UI treats it as mandatory so I thought that was true on the backend as well.

Reply:
I included it just to be more correct.